include/sys/vm_object.h

```diff
@@ -3,14 +3,18 @@
 
 #include <sys/vm.h>
 #include <sys/vm_pager.h>
+#include <sys/mutex.h>
+#include <sys/refcnt.h>
 
 /* At the moment assume object is owned by only one vm_map */
 typedef struct vm_object {
+  mtx_t mtx;
   vm_pagelist_t list;
   vm_pagetree_t tree;
   size_t size;
   size_t npages;
   vm_pager_t *pager;
+  refcnt_t ref_counter;
 } vm_object_t;
```

Comment:
Which fields are protected by `mtx`? Please add comment.

Reply:
Refer to `include/sys/proc.h` for examples.

Reply:
Done